### DISCUSSION TITLE
Shiny.GenerateStaticClasses in a MAUI proyect

### DISCUSSION THREAD

**Discussion by AngeloLazzari** [shinyorg/shiny#1506] [Q&A]
Hi, we are migrating our Xamarin app ( that uses Shiny ) to a new MAUI app, and we want to still have shiny with us. 
From the updated to MAUI Assembly.info file we have this 
`using Microsoft.Maui.Controls.Xaml;

[assembly: XamlCompilation(XamlCompilationOptions.Compile)]
[assembly: Shiny.GenerateStaticClasses("LogikoApp")]`

but the compiler says that no GenerateStaticClasses is still available, what are we doing wrong? Thank you very much

**Comment by aritchie** [comment]
It is long gone.  The nuget `Shiny` is deprecated.  Please use https://shinylib.net/client/appbuilder/ to generate the setup cases

**Comment by AngeloLazzari** [comment]
Thank you very much... so now, what is the right approach? I tell you this because we have a startup.cs class like this:
![startup](https://github.com/user-attachments/assets/28f8965f-dd3f-4e64-9762-71193ec0c63e)
but now we have an errore like: 
ShinyStartup could could not be found .... what we are missing now? thanks

**Reply by aritchie** [reply]
The link I gave you shows you all of the wire ups and boilerplate.  There are samples linked in the docs and in the readme of this app.  Please use them

**Comment by AngeloLazzari** [comment]
Hi Allan, thanks for your reply I am reading and learning about your great shiny .NET.

Just one more thing that I cannot find a solution: in the xamarin app we were calling this
` await Task.Run(() => Shiny.Hosting.Host.GetService<IJobManager>().Register(new JobInfo("SincroJob", typeof(SincroJob), false) 
 {
     // PeriodicTime = new TimeSpan(1, 0, 0),
     // Repeat = true // this is the default
 }));
`
now I hade to comment the PeriodicTime and Repeat because there are not existing anymore, which way I have to set them? 
thank you very much!

**Reply by dbSoft-SoftwareSolutions** [reply]
We have the same issue

**Reply by aritchie** [reply]
They don't exist anymore.  Please spend a little bit of effort looking at the source code in the future.

**Reply by dbSoft-SoftwareSolutions** [reply]
Are you mean this url [](https://shinylib.net/) ?? There is not periodic time ? Also it is not very clear how you can repeat a job?

**Reply by aritchie** [reply]
These jobs are periodic, not scheduled.  The OS controls it.  Jobs repeat automatically
